tests/helpers.py

```diff
@@ -1572,3 +1572,32 @@ def wait_for_ct_pod_recovery():
         else:
             return False
     return True
+
+
+def label_worker_node(node_list, label_key, label_value):
+    """
+    Function to label worker node for running app pods on specific worker nodes.
+
+    Args:
+        node_list (list): List of node name
+        label_key (str): Label_key to be added in worker
+        label_value (str): Label_value
+    """
+    ocp_obj = OCP()
+    for node in node_list:
```

Comment:
Do we want to really run this command that many times as we have number of nodes? This command can run as single command on all the nodes. You just need to do something like this:
```python
out = ocp_obj.exec_oc_cmd(command=f"label node {" ".join(node_list)} {label_key}={label_value}", out_yaml_format=False)
```

Reply:
updated the patch, resolving comment.